packages/plugins/minos-discovery-kong/minos/plugins/kong/client.py

```diff
@@ -1,15 +1,56 @@
+from __future__ import (
+    annotations,
+)
+
+from abc import (
+    ABC,
+)
+from datetime import (
+    datetime,
+    timedelta,
+)
 from uuid import (
     UUID,
 )
 
 import httpx as httpx
+import jwt
+from pytz import (
+    utc,
+)
+
+from minos.common import (
+    Config,
+    SetupMixin,
+)
 
 
-class KongClient:
+class KongClient(ABC, SetupMixin):
```

Comment:
I think `KongClient` is not an abstract class.
```suggestion
class KongClient(SetupMixin):
```

Reply:
Done!